core/metrics/accumulator.go

```diff
@@ -0,0 +1,120 @@
+package metrics
+
+import (
+	"bytes"
+	"fmt"
+	"time"
+
+	"github.com/benbjohnson/clock"
+	"github.com/samber/lo"
+)
+
+type Accumulator struct {
+	clock              clock.Clock
+	updated            time.Time
+	posMeter, negMeter *float64 // kWh
+	Pos                float64  `json:"pos"` // kWh
+	Neg                float64  `json:"neg"` // kWh
```

Comment:
Naming: Pos/Neg ist sehr abstrakt/technisch. Um hier ne saubere Datenbasis zu bekommen werden wir ja auch bei den den Netz-/Batteriezählern das jeweils andere Meter hinzufügen müssen (zumindest mittelfristig). Wie würden wir denn das nennen (`energy[In|Out]`, `import|export`, `energy & energyReverse`)? Ich fänd gut, wenn wir das Naming dann hier bei den Metriken "kompatibel" haben.

Reply:
> Naming: Pos/Neg ist sehr abstrakt/technisch.

Es ist auch eine Datenbanktabelle! Das Naming entspricht 1:1 der evcc Logik der Energieflüsse. Ich wollte da keine neuen Begriffe erfinden. Import/Export sind mindestens bei der Batterie zweideutig.

Was wir aktuell übrigens (leider) nicht können ist hier mit Zählerständen zu agieren da wir die für 2-Richtungszähler mangels APIs nicht kennen sondern immer nur eine Richtung ausgeprägt haben...